tests/e2e.rs

```diff
@@ -836,18 +1115,21 @@ async fn test_e2e_ukt_sync() {
 	let user = user.expect("could not find user");
 	assert_eq!(user.user_name, "delete_me@famedly.de");
 
-	config.perform_sync().await.expect("syncing failed");
+	perform_sync(&config).await.expect("syncing failed");
 
 	let user = zitadel.get_user_by_login_name("delete_me@famedly.de").await;
 	assert!(user.is_err_and(|error| matches!(error, ZitadelError::TonicResponseError(status) if status.code() == TonicErrorCode::NotFound)));
 }
 
+// Currently fails because CSV uses non-hex encoded IDs, need to think
+// about how to fit this into the overall workflow
+#[ignore]
```

Comment:
Not working *yet*

Reply:
I suppose the CSV backend should simply also hex-encode any IDs, and we simply make this an assumption of all sources so that we can rely on this being correct all the time?

Reply:
Yes, all IDs should go through the encoding